pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
```

Comment:
What is the purpose of this new file?

Reply:
its for managing workspaces in pnpm. it allows us to define the different packages in our monorepo and install dependencies of both packages in a node_modules folder